training_pipeline/pipeline/pipeline.py

```diff
@@ -62,7 +62,7 @@ def create_pipeline(
     statistics_gen = StatisticsGen(examples=example_gen.outputs["examples"])
     components.append(statistics_gen)
 
-    schema_gen = SchemaGen(statistics=statistics_gen.outputs["statistics"])
+    schema_gen = tfx.components.ImportSchemaGen(schema_file=schema_path)
```

Comment:
Should we import `ImportSchemaGen` at the top for consistency in style?

Reply:
That is what I thought initially, but it throws an error not to be able to imports with `from tfx.components import ImportSchemaGen`.

Reply:
That is what I thought initially, but it throws an error not to be able to imports with `from tfx.components import ImportSchemaGen`.